lib/web_ui/lib/src/engine/bitmap_canvas.dart

```diff
@@ -902,3 +1097,162 @@ String _cssTransformAtOffset(
   return matrix4ToCssTransform(
       transformWithOffset(transform, ui.Offset(offsetX, offsetY)));
 }
+
+/// JS Interop helper for webgl apis.
+class _GlContext {
```

Comment:
Shouldn't this use dart:web_gl instead? https://api.dartlang.org/dev/2.6.0-dev.7.0/dart-web_gl/dart-web_gl-library.html

Reply:
Our goal is to have small compiled code size, so we try to avoid using convenience libraries. We might do the same with `dart:html` in the future. It's just that right now it's too damn convenient :)

Reply:
Due to code size/treeshaking. Should reconsider once dart:html/web_gl library stops pulling in all classes.

Reply:
Gotcha! Marking as resolved